### ISSUE TITLE
Więcej tagów

### ISSUE THREAD

**Issue by mariuszkarowski** [mariuszkarowski/busy#9]
#### Expected behavior

Więcej tagów button linkowałby do konta które publikowałoby np. raz dziennie, zestawienia tagów polskich. Tak jak robi to @jacekw https://pl.network/@jacekw/najlepiej-wynagradzani-autorzy-w-popularnych-tagach-sierpien-2018
Każdy tag w takim zestawieniu powinien być w formie linku.

Ja takie konto mogę założyć i przekazać klucze osobie która prowadziłaby bota (jak rozumiem to się robi automatycznie?)

**Comment by fervi** [comment]
W sensie chcesz by bot publikował np. raz dziennie listę wszystkich znanych polskich tagów?

**Comment by mariuszkarowski** [comment]
Tak, taki ranking pod względem zarobków, na przykład. Pod tagami można by dodać linki do poradników, ciekawych akcji itp. Co o tym myślisz?

**Comment by fervi** [comment]
To nie jest złe, tylko nie wiem czy chcesz pisać coś od zera (można) czy wykorzystać np. 
https://steemweb.pl/categories

**Comment by mariuszkarowski** [comment]
Ja chciałbym żeby to było TOP 100 tagów pod względem zarobków za ostatnie 30 dni. Dokładnie tak jak robi to jacekw, tylko on publikuje takie zestawienie raz na miesiąc a my moglibyśmy uaktualniać taką listę codziennie.

**Comment by fervi** [comment]
Ok, czekamy na najnowszego Hard Forka i będziemy pracować nad tym
